articles/tables.md

```diff
@@ -20,727 +20,191 @@ category: howto
-df <- as.data.frame(results)
-df['purchased'] <- c('yes', 'yes')
-table <- synStore(Table("syn7264701", df))
-{% endhighlight %}
-{% endtab %}
-
-{% tab Web %}
-To modify row entries, click on the **Edit icon** to the right of the **Query** button. In the resulting pop-up, you can adjust each entry as you please. In this example, the first and last entry of row two have been updated.
-<br>
-<img id="image" src="/assets/images/table_modify_rows.png">
-{% endtab %}
+# Using Table Facets
+The faceted navigation on `Tables` (also known as **simple search**) can be used to simplify your search without having to use SQL-like queries. Simple search uses radio buttons and sliders to show all available facets in a menu to the left of the `Table` whereas advanced search employs a SQL-like query to filter the `Table`. To use table facets, navigate to a `Table` or a `File View`. Simple and advanced search both allow you to query for features of interest in a`Table` using different methods.
 
-{% endtabs %}
+## Set Facets
+In order to use simple search, you must first set columns to be facets in the schema editor. Select **Schema** in the upper right of your table and click on **Edit Schema**. In the resulting pop-up, select **Values** or **Range** from the dropdowns under the **Facet** option. **Values** can be thought of as categories whereas **Range** is a date or number.
 
-<br>
+<img id="imageLg" src="/assets/images/set_facets.png">
 
-### Deleting the whole table
+{% include note.html content="If you change Column Type in the schema, you have to set its facet selection again." %}
 
-**Delete the entire table**
+To see all the facets, click on **Show simple search** found above the SQL-query bar:
 
-{% tabs %}
+<img id="imageLg" src="/assets/images/show_simple_search.png">
 
-{% tab Python %}
-{% highlight python %}
-#Deleting the schema deletes the whole table and all rows
-syn.delete(schema)
-{% endhighlight %}
-{% endtab %}
+Select the features you are interested in to filter the table.
 
-{% tab R %}
-{% highlight r %}
-#Deleting the schema deletes the whole table and all rows
-synDelete(schema)
-{% endhighlight %}
-{% endtab %}
+<img id="imageLg" src="/assets/images/simple_search.png">
 
-{% tab Web %}
-To delete the entire Table, click on **Tools** and then select **Delete Table** from the resulting dropdown.
-<br>
-<img id="image" src="/assets/images/delete_table.png">
-{% endtab %}
+## Toggling between Simple and Advanced Search
+You can toggle from the simple search to the advanced search without losing the query results. For example, if in the simple search you had selected treatmentArm `A`, age of `23:64`, and gender as `female`, the query will be preserved in the advanced search bar. However, this is unidirectional because the advance search allows parameters that are not available with facets. Therefore switching from advanced to simple search will result in resetting the search query.
```

Comment:
```suggestion
You can toggle from the simple search to the advanced search without losing the query results. For example, if in the simple search you had selected treatmentArm `A`, age of `23:64`, and gender as `female`, the query will be preserved in the advanced search bar. However, this is unidirectional because the advanced search allows parameters that are not available with facets. Therefore switching from advanced to simple search will result in resetting the search query.
```